tools/lsp/main.rs

```diff
@@ -271,7 +272,7 @@ fn handle_request(
         // representation.
         let requested_color = params.color;
 
-        let color_literal = if (requested_color.alpha - 1.).abs() > 10.0 * f32::EPSILON {
+        let color_literal = if requested_color.alpha.approx_eq(&1.) {
```

Comment:
This was originally `!= 1`.

```suggestion
        let color_literal = if !requested_color.alpha.approx_eq(&1.) {
```

Reply:
I'd go as far as simply say 
```suggestion
        let color_literal = if requested_color.alpha < 1. {
```

Reply:
Good catch!

I'll go with @tronical s approach: We want that path to be taken, even if we just have 0.9999999 :-)

These values are potentially the result of a calculation with some rounding error.